.vsts-pipelines/templates/default-build.yml

```diff
@@ -48,6 +48,7 @@ parameters:
   afterBuild: []
   codeSign: false
   variables: {}
+  runCodeQL3000: false
```

Comment:
nit: No reason for this really. Could use `parameters.variables` in ci-public.yml (odd we named the file that way, given it's used for official builds too)

Reply:
I think this is more readable